### ISSUE TITLE
Deprecating and removing Stanford API support 

### ISSUE THREAD

**Issue by alvations** [nltk/nltk#1892]
Stanford CoreNLP has released their official Python wrapper https://github.com/stanfordnlp/python-stanford-corenlp 

Should we remove the support for the Stanford API in NLTK and replace it with `NotImplementedError`?

**Comment by stevenbird** [comment]
Good idea, but I think we should deprecate it first, and remove it in the next major release.
